core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -37,6 +37,8 @@ class EmbeddedZookeeper() {
   factory.startup(zookeeper)
   val port = zookeeper.getClientPort()
 
+  println("Starting embedded zookeeper on port " + port + " using directories " + snapshotDir + "," + logDir)
```

Comment:
I had a look at the test output from the failing build run and it looked like only warnings and errors were being logged. And there were other tests doing println, so I guessed that would be fine.